mysql-test/main/change_master_default.test

```diff
@@ -0,0 +1,145 @@
+# MDEV-28302: Test how `CHANGE MASTER [named]` reads server
+# options for its default values and the lack of those options.
+# The test creates multiple CHANGE MASTER connections,
+# where the defaultable fields of a connection are:
+# * left unset
+# * set to `DEFAULT`
+# * configured with values
+
+# only need CHANGE MASTER and IS.slave_status
+--source include/have_binlog_format_mixed.inc
+
+CREATE PROCEDURE show_defaultable_fields()
+SELECT connection_name,
+  connect_retry,
+  master_ssl_allowed,
+  master_ssl_ca_file,
+  master_ssl_ca_path,
+  master_ssl_cert,
+  master_ssl_cipher,
+  master_ssl_key,
+  `master_ssl_verify_server_cert`, # MDEV-38194
+  master_ssl_crl,
+  master_ssl_crlpath,
+  using_gtid,
+  master_retry_count,
+  slave_heartbeat_period
+FROM information_schema.slave_status ORDER BY connection_name;
+
+CHANGE MASTER 'unset' TO master_host='127.0.1.1';
+
+CHANGE MASTER 'defaulted' TO
+  master_connect_retry= DEFAULT,
+  master_ssl= DEFAULT,
+  master_ssl_ca= DEFAULT,
+  master_ssl_capath= DEFAULT,
+  master_ssl_cert= DEFAULT,
+  master_ssl_cipher= DEFAULT,
+  master_ssl_key= DEFAULT,
+  master_ssl_verify_server_cert= DEFAULT,
+  master_ssl_crl= DEFAULT,
+  master_ssl_crlpath= DEFAULT,
+  master_use_gtid= DEFAULT,
+  master_retry_count= DEFAULT,
+  master_heartbeat_period= DEFAULT,
+  master_host= '127.0.1.2';
+
+# This subject does not stay constant.
```

Comment:
I don't know what this comment is trying to say

Reply:
Besides demonstrating nameless CHANGE MASTER, `DEFAULT` will also overwrite these values, thus it is not named as “specified”.

… yeah, I’m discarding this pedantic comment.